config/locales/en/course/user_registrations.yml

```diff
@@ -6,6 +6,9 @@ en:
         register: 'Register'
         deregister: 'Cancel Registration Request'
         enter_course: 'Enter Course'
+        new_enrol_request_text_html: >
+          If you don't have an invitation code, %{click_here} to send an enrol request
```

Comment:
"If you don't have a code, %{click_here} to request enrolment in this course."

Because there are course registration codes and individual user invitation codes. Another possibility is to standardise the naming of the codes.